### PR TITLE
both assignments are complete

### PR DESCRIPTION
**Pull request by ThePro1604**
i also did the bonus phone display thing in the second assignment

### CODE REVIEW

File: First-Html-page/index.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+/*body {*/
+/*    margin: 0;*/
+/*}*/
+
+/*html, body {*/
+/*    margin: 0;*/
+/*    height:100%;*/
+
+/*}*/
```
Comment (Tamir198) [inline First-Html-page/index.css:21]: If you don't need this code, delete it but don't comment it.

File: First-Html-page/index.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+h2 {
+    font-family: Montserrat;
+    color: #66BFBF;
+    letter-spacing: .08rem
+}
+
+
+h3, h4, h5, h6 {
+    font-family: Montserrat;
```
Comment (Tamir198) [inline First-Html-page/index.css:74]: You can include h2 in here as well, you are giving it the same font

File: First-Html-page/index.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    letter-spacing: .09rem
+
+}
+
+
+.splitscreen {
+    display: flex;
+}
+
+.splitscreen .left{
```
Comment (Tamir198) [inline First-Html-page/index.css:106]: I think you can find a better name for your left class name. Try something more clear - left if very generic.

File: First-Html-page/personal-site-challenge.html [hunk 4]
```diff
@@ -5,25 +5,84 @@
...
-        If you love coding as much as I do.
-        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci consectetur quisquam deleniti, molestiae reiciendis atque hic blanditiis suscipit voluptates harum ducimus, rerum alias! Quos cum, sequi molestiae itaque id sunt!
-        LinkedIn
-        Twitter
-        Website
-        © DevClub
+    <header>
+        <div class="overlay">
+            <img alt="top-cloud" style="margin-right: -70%; margin-top: 40px; margin-bottom: 50px" src="asset/cloud.png" />
+            <h1 style="font-size: 50px">I'm a Dev Club Member.</h1>
```
Comment (Tamir198) [inline First-Html-page/personal-site-challenge.html:17]: why inline style and not css styling?

**Review by Tamir198 — COMMENTED** [review]
Some general things I saw repeating in your project:

- don't comment code that you don't need, delete it instead.
- don't mix inline styles and styling thing inside CSS files
- work on better class names, for example - using `body` on a `div` inside `<body>` is a bit confusing

**Comment by ThePro1604** [comment]
not sure ill be able to finish animation on time

**Pull request closed without merging**